_posts/scyrill/11-23-2021-scyrill-project-update.md

```diff
@@ -0,0 +1,158 @@
+---
```

Comment:
In the file title above, you may want to change the file name to match the date structure in [your other post files](https://github.com/inf380p/inf380p.github.io/tree/production/_posts/scyrill). The date syntax (`YYYY-MM-DD-`) is important for the code that builds the website, so your post may not show up where you think it should if it isn't written exactly like it expects.